proto/umee/leverage/v1beta1/query.proto

```diff
@@ -102,6 +107,19 @@ message QueryLendAPYResponse {
   [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec", (gogoproto.nullable) = false];
 }
 
+// QueryMarketSizeRequest defines the request structure for the Market Size in USD
+// gRPC service handler.
+message QueryMarketSizeRequest {
+  string denom = 1;
+}
+
+// QueryMarketSizeResponse defines the response structure for the Market Size in USD
+// gRPC service handler.
+message QueryMarketSizeResponse {
+  string marketSizeUSD = 1
```

Comment:
proto fields are snake-cased :)

```suggestion
  string market_size_usd = 1
```

Reply:
this way the attribute stays `MarketSizeUsd` instead of `MarketSizeUSD` is that okay?